components/SearchResultsList.vue

```diff
@@ -10,24 +10,35 @@
                     class="hamburger-list-icon fill-primary/50 h-5 w-5 self-center group-hover:fill-primary-text-inverted/50">
                     <use xlink:href="../assets/images/hamburger-list-icon.svg#hamburger-list-icon-svg" />
                 </svg>
-                <span class="text-primary-text/50 pl-1 font-bold self-center group-hover:text-primary-text-inverted/50">{{
-                    $t('searchResultsList.filters') }}</span>
+                <span
+                    class="text-primary-text/50 pl-1 font-bold self-center group-hover:text-primary-text-inverted/50">{{
+                        $t('searchResultsList.filters') }}</span>
```

Comment:
Is this part just a formatting change? If so, let's leave it out.

Reply:
Hmm, I don't remember intentionally do this. This may have occurred via prettier or the linter? I will verify